recipes/proxy/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.52.0"
+
+
+class ProxyConan(ConanFile):
+    name = "proxy"
+    description = "Proxy: Next Generation Polymorphism in C++"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/proxy"
+    topics = ("runtime-polymorphism", "polymorphism", "duck-typing", "metaprogramming", "header-only")
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "11",
+            "clang": "12",
+            "apple-clang": "13",
```

Comment:
```suggestion
            "apple-clang": "14",
```

The std::has_single_bit should be available, but I guess it requires some update. Let's keep 14, so we are safe. Tested locally with Apple Clang 15 worked fine.


https://en.cppreference.com/w/cpp/compiler_support#C.2B.2B20_library_features